wordpress-mobile/gutenberg-mobile
# Move BlockPicker to gutenberg as Inserter in @wordpress/block-editor

Step 3 of https://github.com/wordpress-mobile/gutenberg-mobile/issues/958
This is the next step in our port of the BlockList to gutenberg.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/16114

### Testing Instructions
- Boot the app on iOS and Android
- Check that the block picker still works as expected

Update release notes:
